docs/Concepts/Privacy/Privacy-Groups.md

```diff
@@ -1,76 +1,86 @@
-description: Privacy
+description: Privacy groups
 <!--- END of page meta data -->
 
-# Privacy Groups 
+# Privacy groups
 
-A privacy group is a group of nodes identified by a unique privacy group ID by Orion. Each private transaction is 
-stored in Orion with the privacy group ID.
+A privacy group is a group of nodes identified by a unique privacy group ID by Orion. Each private
+transaction is stored in Orion with the privacy group ID.
 
-The Besu nodes maintain the public world state for the blockchain and a private state for each privacy group. 
-The private states contain data that is not shared in the globally replicated world state. 
+The Besu nodes hold the public world state for the blockchain and a private state for each privacy
+group. The private states contain data that is not shared in the globally replicated world state.
 
-## Privacy Types 
+## Privacy types
 
-Besu implements two types of privacy: 
+Besu implements two types of privacy:
 
-* EEA-compliant privacy - private transactions include `privateFor` as the recipient.
-* Besu-extended privacy - private transactions include `privacyGroupId` as the recipient.
+* Enterprise Ethereum Alliance privacy, where private transactions include `privateFor` as the
```

Comment:
I think we still need to include the EEA acronym here to make the link to the API name. Maybe Enterprise Ethereum Alliance (EEA) privacy

Reply:
Acronym included.